docs/code_navigation/how-to/index_other_languages.mdx

```diff
@@ -4,48 +4,43 @@ This guide is meant to provide instructions for how to enable precise code navig
 The general steps for enabling precise code navigation are as follows:
 
 1. Install Sourcegraph CLI
-2. Install LSIF Indexer
-3. Generate LSIF data
-4. Upload LSIF data
+2. Install SCIP Indexer
+3. Generate SCIP data
+4. Upload SCIP data
 
 ### 1. Install Sourcegraph CLI
 
-The [Sourcegraph CLI](/cli/) is used for uploading LSIF data to your Sourcegraph instance (replace `linux` with `darwin` for macOS):
+The [Sourcegraph CLI](/cli/) is used for uploading SCIP data to your Sourcegraph instance (replace `linux` with `darwin` for macOS):
 
 ```
 $ curl -L https://sourcegraph.com/.api/src-cli/src_linux_amd64 -o /usr/local/bin/src
 $ chmod +x /usr/local/bin/src
 ```
 
-### 2. Install LSIF indexer
+### 2. Install SCIP indexer
 
-An LSIF indexer is a command line tool that performs code analysis of source code and outputs a file with metadata containing all the definitions, references, and hover documentation in a project in LSIF. The LSIF file is then uploaded to Sourcegraph to power code navigation features.
+An SCIP indexer is a command line tool that performs code analysis of source code and outputs a file with metadata containing all the definitions, references, and hover documentation in a project in SCIP. The SCIP file is then uploaded to Sourcegraph to power code navigation features.
 
 Install the indexer for the required programming language of your repository by following the instructions in the indexer's README:
 
-  1. [C++](https://github.com/sourcegraph/lsif-cpp)
-  1. [Dart](https://github.com/sourcegraph/lsif-dart)
-  1. [Go](https://github.com/sourcegraph/lsif-go)
-  1. [Haskell](https://github.com/mpickering/hie-lsif)
-  1. [Java](https://github.com/sourcegraph/scip-java)
-  1. [JavaScript/TypeScript](https://github.com/sourcegraph/scip-typescript)
-  1. [Jsonnet](https://github.com/sourcegraph/lsif-jsonnet)
-  1. [Python](https://github.com/sourcegraph/scip-python)
-  1. [OCaml](https://github.com/rvantonder/lsif-ocaml)
-  1. [Scala](https://github.com/sourcegraph/lsif-semanticdb)
+  1. [C++](https://github.com/sourcegraph/scip-cpp) 
```

Comment:
@varungandhi-src  Not sure if we want this or if we still want to reference the LSIF ones

Reply:
Instead of repeating the names of some indexers here, I think it would be better to link to the indexers page: `/code_navigation/references/indexers`.

Aside: The C++ indexer is called scip-clang.